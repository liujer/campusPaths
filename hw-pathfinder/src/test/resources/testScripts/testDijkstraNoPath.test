# Attempt to find a path that does not exist with Dijkstra's
# algorithm
CreateGraph A
AddNode A one
AddNode A two
AddNode A three

AddEdge A one two 2

FindPath A one three