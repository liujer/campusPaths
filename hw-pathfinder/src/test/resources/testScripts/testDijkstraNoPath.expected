# Attempt to find a path that does not exist with Dijkstra's
# algorithm
created graph A
added node one to A
added node two to A
added node three to A

added edge 2.000 from one to two in A

path from one to three:
no path found